gemfiles/Gemfile.public_suffix_2

```diff
@@ -0,0 +1,34 @@
+source 'https://rubygems.org'
```

Comment:
Just to clarify, this `Gemfile` is only for testing purposes? If so, would it make more sense to name it in such a way that that's obviously, or perhaps include a comment to that effect?